packages/hub/src/app/users/[username]/models/[slug]/ModelLayout.tsx

```diff
@@ -126,48 +161,39 @@ export const ModelLayout: FC<Props> = ({ username, slug, children }) => {
 
   return (
     <EntityLayout
-      slug={slug}
-      username={username}
-      homepageUrl={modelRoute({ username, slug })}
+      nodes={entityNodes(username, slug, variableName)}
       isFluid={true}
       headerChildren={
         <>
-          {Boolean(model.currentRevision.relativeValuesExports.length) && (
-            <Dropdown render={({ close }) => dropDown(close)}>
-              <div className="flex items-center rounded cursor-pointer hover:bg-white px-2 py-1 select-none text-sm">
-                <ScaleIcon size={16} className="text-gray-500" />
-                <TriangleIcon
-                  size={7}
-                  className="rotate-180 ml-2 text-slate-400"
-                />
-              </div>
-            </Dropdown>
-          )}
-          <StyledTabLink.List>
-            <StyledTabLink
-              name="Editor"
+          <EntityTab.List>
+            <EntityTab.Link
+              name="Code"
+              icon={CodeBracketIcon}
               href={modelRoute({ username, slug })}
             />
-            {model.currentRevision.relativeValuesExports.length ? (
-              <StyledTabLink
-                name="Viewer"
-                href={modelViewRoute({ username, slug })}
-                selected={(pathname, href) =>
-                  pathname === href ||
-                  pathname.startsWith(
-                    modelRoute({ username, slug }) + "/relative-values"
-                  )
-                }
-              />
-            ) : null}
-            <StyledTabLink
+            {Boolean(model.currentRevision.relativeValuesExports.length) && (
+              <Dropdown render={({ close }) => dropDown(close)}>
```

Comment:
Not very important, and belongs to your previous PR, but here's an alternative way to type this callback:
```
const renderExportsDropdown: Parameters<typeof Dropdown>[0]["render"] = ({ close }) => (
```

And later:
```
<Dropdown render={renderExportsDropdown}>
```

Reply:
I spent a bit of time on that. That's neat, but feels too complicated for the benefit in this one. Good to know for the future though!